### PR TITLE
- applying https://github.com/freeorion/freeorion/pull/833 as patch

### PR DESCRIPTION
**Pull request by st-pa-fo**
cause i'd like to see old issues closed. see also http://www.freeorion.org/forum/viewtopic.php?f=15&t=9746 and https://github.com/freeorion/freeorion/pull/833 for more details, e.g. it was mentioned there, that some AI adjustments might need to be made

### CODE REVIEW

**Comment by st-pa-fo** [comment]
ahhrgh and i forgot the checksum thing and string tables test again .. one more commit to fix that

File: default/stringtables/en.txt [hunk 1]
```diff
@@ -4442,6 +4442,7 @@ Locked
 ALL_POLICY_AVAILABILITY_FILTERS_BLOCKING_PROMPT
 Toggle the category and availability filters to show policies.
 
+
```
Comment (geoffthemedio) [inline default/stringtables/en.txt:4445]: If you have a fix like this, it's a lot better to put it into a separate pull request that fixes just that, so I can just accept it. If it's added to an unrelated pull request like this, I have to separately pull and [commit it](https://github.com/freeorion/freeorion/commit/bddeabbcbabfadb50946540f94b1c5845aee3e3a) or wait until the rest of the pull request is OK...

**Comment by geoffthemedio** [comment]
Given the current state of the AI, needing some AI updates for this probably isn't a problem now.

Oddly, I can't seem to check it out though... git says
````
git pull git://github.com/st-pa-fo/freeorion.git 'stpa-rebumping-#833'
fatal: couldn't find remote ref 'stpa-rebumping-#833'
````
despite that being the command line github reccommends.

**Comment by geoffthemedio** [comment]
If updating someone else's patch like this, it's preferable to retain their authorship of it, eg. https://github.com/freeorion/freeorion/pull/3649

**Comment by Vezzra** [comment]
Superseded by #3649

**Pull request closed without merging**